test/runtests.jl

```diff
@@ -8,4 +8,8 @@ include("split.jl")
 include("destruct.jl")
 include("utils.jl")
 
+if VERSION>=v"1.8"
+    include("flatten_try.jl")
+end
```

Comment:
I don't think we need to / should special-case here. The PR will work on older versions of Julia, and it's nice to fix #196 there too.

Reply:
The `try catch else` is not valid syntax for earlier versions of Julia. That is why I have some tests copied in utils.

Reply:
The tests in utils will execute on earlier versions of julia and take care of the examples that do not contain `else`

Reply:
👍 Maybe just a `# because of new try/else syntax` comment then please.